docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -4,127 +4,111 @@ description: Provides instructions for upgrading Teleport Cloud agents that run
-   ```
-
-1. Validate the updater is running properly by checking if its pod is ready:
-
-   ```code
-   $ kubectl -n <Var name="teleport" /> get pods
-   NAME                               READY   STATUS    RESTARTS   AGE
-   my-agent-0                         1/1     Running   0          14m
-   my-agent-1                         1/1     Running   0          14m
-   my-agent-2                         1/1     Running   0          14m
-   my-agent-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
-   ```
-   
-   And by consulting its logs:
-   
-   ```code
-   $ kubectl -n <Var name="teleport" /> logs deployment/<Var name="teleport-agent" />-updater
-   2023-04-28T13:13:30Z	INFO	StatefulSet is already up-to-date, not updating.	{"controller": "statefulset", "controllerGroup": "apps", "controllerKind": "StatefulSet", "StatefulSet": {"name":"my-agent","namespace":"agent"}, "namespace": "agent", "name": "my-agent", "reconcileID": "10419f20-a4c9-45d4-a16f-406866b7fc05", "namespacedname": "agent/my-agent", "kind": "StatefulSet", "err": "no new version (current: \"v12.2.3\", next: \"v12.2.3\")"}
-   ```
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
 
-The updater is a controller that periodically reconciles expected Kubernetes
-resources with those in the cluster. The updater executes a reconciliation loop
-every 30 minutes or in response to a Kubernetes event. If you don't want to wait
-until the next reconciliation, you can trigger an event. Any deployment update
-will send an event, so the updater can be triggered by annotating the resource:
+Add the following chart values to your existing agent `values.yaml` to enable
+the automatic updater:
+
+```yaml
+updater:
+    enabled: true
+```
+
+Upgrade the Helm chart release with the new values by running `helm upgrade`:
 
 ```code
-$ kubectl -n <Var name="teleport" /> annotate statefulset/<Var name="teleport-agent" /> 'debug.teleport.dev/trigger-event=1'
+$  helm -n <Var name="teleport" /> upgrade <Var name="teleport-agent" /> teleport/teleport-kube-agent  \
+--values=values.yaml \
+--version=<Var name="(=cloud.version=)" />
 ```
 
-To suspend automatic updates for an agent, annotate the agent deployment with
-`teleport.dev/skipreconcile: "true"`, either by setting the
-`annotations.deployment` value in Helm, or by patching the deployment directly
-with `kubectl`.
+## Manual updates
+
+[Automatic updates](#automatic-updates) is the recommended update method for Teleport
+agents. If you choose to opt out of automatic updates, you will need to be responsible
+for keeping your Teleport agents up to date with the version of Teleport Cloud.
+
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+Upgrade the Helm release:
 
-## ArgoCD deployments
+```code
+$ helm -n <Var name="teleport" /> upgrade <Var name="teleport-agent" /> teleport/teleport-kube-agent \
+--values=values.yaml \
+--version=<Var name="(=cloud.version=)" />
+```
+
+## FAQ
```

Comment:
I would rename this H2 and move the "Manual updates" section within it, since this H2 includes instructions for nonstandard situations. Maybe we could call this section "Edge cases" or similar?